hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java

```diff
@@ -326,6 +329,10 @@ private void initSCM() throws IOException {
     scmStore.setScmId(scmId);
     // writes the version file properties
     scmStore.initialize();
+    if (!SCMHAUtils.isSCMHAEnabled(conf)) {
```

Comment:
Question: Why do we need this step here?

Reply:
This step is required bcoz, otherwise, scm start won't elect a ratis leader as this is initializing scm version file without calling explicit scmInit.